ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -302,47 +303,45 @@ def aws_cleanup():
             over the value provided to --hours). Note: if you want to skip
             cleanup of a cluster entirely you can use 'never' for the hour.
             Example: --prefix foo:24 --prefix bar:48 --prefix foobar:never
-            """
+            """,
     )
     parser.add_argument(
-        '--force',
-        action='store_true',
+        "--force",
+        action="store_true",
         required=False,
         help="""
             Force cluster cleanup.
             User will not be prompted for confirmation.
             WARNING: this utility is destructive, only use this option if
             you know what you are doing.
-            """
+            """,
     )
     args = parser.parse_args()
 
     if not args.force:
         confirmation = input(
-            'Careful! This action could be highly destructive. '
-            'Are you sure you want to proceed? '
-        )
-        assert confirmation == defaults.CONFIRMATION_ANSWER, (
-            "Wrong confirmation answer. Exiting"
+            "Careful! This action could be highly destructive. "
+            "Are you sure you want to proceed? "
         )
+        assert (
+            confirmation == defaults.CONFIRMATION_ANSWER
+        ), "Wrong confirmation answer. Exiting"
 
     prefixes_hours_to_spare = defaults.CLUSTER_PREFIXES_SPECIAL_RULES
 
     if args.prefix:
         for prefix, hours in args.prefix:
             logger.info(
-                "Adding special rule for prefix '%s' with hours %s",
-                prefix, hours
+                "Adding special rule for prefix '%s' with hours %s", prefix, hours
```

Comment:
pleas use f string

Reply:
We suggest using the logger to format strings like this in our [coding guidelines](https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md).